src/BenchmarksClient/Startup.cs

```diff
@@ -34,6 +35,13 @@ public void ConfigureServices(IServiceCollection services)
         public void Configure(IApplicationBuilder app)
         {
             app.UseMvc();
+
+            app.Map("", builder => 
```

Comment:
Why was this added?

Reply:
Diagnistics page, just to be sure the app is up and also to be sure the url is valid when configuring docker.

Reply:
Please add a comment explaining this.